Monika After Story/game/script-topics.rpy

```diff
@@ -10501,4 +10501,159 @@ label monika_hemispheres_nogets_snow:
+    default persistent._mas_pm_wants_children = None
+    # do you want children?
+    
+    default persistent._mas_pm_wants_1_child = None
+    # do you want one child?
+    
+    default persistent._mas_pm_wants_boy = None
+    # do you want a boy?
+    
+    default persistent._mas_pm_wants_girl = None
+    # do you want a girl?
+    
+    default persistent._mas_pm_wants_2_to_3_children = None
+    # do you want two to three children?
+    
+    default persistent._mas_pm_wants_many_children = None
+    # do you want many children?
+    
+    default persistent._mas_pm_wants_children = None
+    # do you not want children?
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="Children",category=['romance','monika','life'],prompt="Having Children",random=True))
+
+label monika_children:
+    if mas_isMoniLOVE(higher=True):
+    m 3rkc "Hey, [player]..."
+    m 3rkbsd "Did you know when couples are close enough they start talking about having children?"
+    m 3rkbfsdlc "O-Of course you knew that, Ahaha~" 
+    m 2dkbfc "..." 
+    m 4rsbfsdla "Just a thought I had, you know..." 
+    show monika 5rsbfsdld at t11 zorder MAS_MONIKA_Z with dissolve
+    m 5rsbfsdld "I was wondering if-" 
+    m 2dsbfc "..." 
+    m 3hubfsdlb "Ahaha. I thought this would be easier to talk about with you..." 
+    m 4etbfc "I guess I'll get to the point."
+    menu:
+        m 2esbfsdlc "When I make it out there to your reality, would you like to have children with me?"
+        "Yes.":
+            $ persistent._mas_pm_wants_children = True
+            m 6sublo "REALLY?!"
+            m 1subltdb "Oh, [player]! You don't know how happy that makes me, my love!"
+            menu:
+                m 3hua "How many kids do you want:"
+                "Just one." 
+                    $ persistent._mas_pm_wants_1_child = True
+                    m 4esa "Yeah, that sounds about right." 
+                    m 6wub "Can you imagine it, [player]?"
+                    m 6hubsa "Just the three of us..." 
+                    m 6hubsa "Living life together." 
+                    m 6hubsa "Helping them grow up."
+                    menu:
+                        m 3eua "What do you think, sweetie? A boy or a girl?"
+                        "Boy": 
+                            $ persistent._mas_pm_wants_boy = True
+                            m 6hubla "A boy would be wonderful." 
+                            m 6hubla "Just imagine it."
+                                if persistent.gender == "M":
+                                    m 4eubla "He starts taking after you a bit."  
+                                    m 4eubld "Tries on all of your clothes." 
+                        "Girl": 
+                            $ persistent._mas_pm_wants_girl = True
+                            m 6hubla "A girl would be wonderful." 
+                            m 6hubla "Just imagine it."
+                                if persistent.gender == "F":
```

Comment:
This `if` block (and it's corresponding `else` block) should move back an indent.